editor/src/components/canvas/canvas-strategies/absolute-move-strategy.ts

```diff
@@ -78,6 +94,13 @@ export const absoluteMoveStrategy: CanvasStrategy = {
           highlightedViews: {
             $set: [],
           },
+          canvas: {
+            controls: {
+              snappingGuidelines: {
+                $set: guidelinesWithSnappingVector, // these will be used by the guideline controls
```

Comment:
Can you create a command for this rather than resorting to the wildcard patch please